ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -469,5 +456,20 @@ class JITModBuilder {
       if (G->hasInitializer())
         NewGV->setInitializer(MapValue(G->getInitializer(), VMap));
     }
+
+    // Ensure that the JITModule has a `!llvm.dbg.cu`.
+    // This code is borrowed from LLVM's `cloneFunction()` implementation.
+    // OPT: Is there a faster way than scanning the whole module?
```

Comment:
I assume we only ever run this in debug mode, so it's probably not worth worrying about. Otherwise, we might be able to inline it into the processing of operands.